src/com/shapestone/files/ReadJsonFile.java

```diff
@@ -0,0 +1,45 @@
+package com.shapestone.files;
+
+import java.io.File;
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class ReadJsonFile {
+	public static void main(String[] args) throws StreamReadException, DatabindException, IOException {
+		ObjectMapper om = new ObjectMapper();
+		Student[] readValue = om.readValue(new File("studentjsonfile.json"), Student[].class);
+
+		System.out.printf("| %-15s | %-15s | %-15s | %-90s |%n", "name", "course", "phone number",
+				"                                             address");
+		System.out.printf("------------------------------------------------------------------------------------"
+				+ "--------------------------------------------------------------%n");
+
+		for (int i = 0; i < readValue.length; i++) {
+
+			// System.out.println(readValue[i].getAddress());
+			System.out.printf("| %-15s | %-15s | %-15s | %-88s|%n", readValue[i].getName(), readValue[i].getCourse(),
+					readValue[i].getPhoneNumber(), readValue[i].getAddress());
```

Comment:
I think address text will be huge here, add more columns like Village, Mandal, State, etc..